### PR TITLE
chore: separate instance handlers

### PR DESCRIPTION
**Pull request by Michsior14**
This separates the handing of messages based on instance type to avoid growing `server.ts` code.
I was not sure about the naming/files structure and will be happy to improve it depending on needs.

Hope this idea will be helpful ;)

### CODE REVIEW

File: lib/node/incoming_message.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { ValueID } from "zwave-js";
+import { IncomingCommandBase } from "../incoming_message_base";
+import { NodeCommand } from "./command";
+
+export interface IncomingCommandNodeBase extends IncomingCommandBase {
+  nodeId: number;
+}
+
+export interface IncomingCommandNodeSetValue extends IncomingCommandNodeBase {
+  command: NodeCommand.setValue;
```
Comment (balloob) [inline lib/node/incoming_message.ts:10]: I am not a fan of this. It just means we are declaring everything in other places and in the end it's unclear what is happening where.

Comment (balloob) [reply lib/node/incoming_message.ts:10]: It's an improvement, we can always move things around later. Let's just go with it .

**Review by balloob — APPROVED** [review]
This looks great. Thanks!

File: lib/node/incoming_message.ts (hunk 1 above)
Comment (Michsior14) [reply lib/node/incoming_message.ts:10]: I've did it only to make sure the property in `NodeResultTypes` matches the command ;) It's easy to misspell it especially when the number of commands grow.

**Pull request merged**